setup.py

```diff
@@ -17,11 +17,14 @@ def get_tag(self):
 # Read the clang-format version from the "single source of truth"
 def get_version():
     with open("clang-format_version.cmake", "r") as version_file:
-        version_line = version_file.read()
-        match = re.match("set\(CLANG_FORMAT_VERSION (.*)\)", version_line)
-        if not match:
-            raise ValueError("Version File not readable")
-        return match.groups()[0]
+        parsed = {}
+        for line in version_file:
+            match = re.match("set\((.*) (.*)\)", line)
+            if len(match.groups()) != 2:
+                raise ValueError("Version File not readable")
+            parsed[match.groups()[0]] = match.groups()[1]
+
+        return f"{parsed['CLANG_FORMAT_VERSION']}.{parsed['CLANG_FORMAT_WHEEL_VERSION']}"
```

Comment:
If the `CLANG_FORMAT_WHEEL_VERSION` is 0 (the first release of the wheel for a particular clang version) maybe the wheel version could be implicit? Are there any side effects for pip installing future versions with wheel-only changes if this were done (@henryiii)?

```suggestion
        if parsed['CLANG_FORMAT_WHEEL_VERSION'] == "0":
            return f"{parsed['CLANG_FORMAT_VERSION']}"
        else:
            return f"{parsed['CLANG_FORMAT_VERSION']}.{parsed['CLANG_FORMAT_WHEEL_VERSION']}"
```

Reply:
No, there shouldn't be a difference. 1.2.3.0 should be identical to 1.2.3.

Reply:
Good point and thanks to @henryiii for confirming.